extensions/vscode/package.json

```
@@ -13,6 +13,7 @@
     "url": "https://github.com/VeryGoodOpenSource/dart_frog"
   },
   "homepage": "https://dartfrog.vgv.dev/",
+  "license": "SEE LICENSE IN LICENSE FILE",
```

As we do have a SPDX recognized license, we can add its identifier:

```suggestion
  "license": "MIT",
```

---

That was my initial commit, but I changed it to forward to LICENSE so if something is ever changed, the LICENSE file is the single point of truth.

Wdyt?

---

@alestiago but I think we need to have a valid license name here right? In case someone (a store) reads this?

---

I think both approaches are valid, but the documentation says the following:

> If you do have a LICENSE file in the root of your extension, the value for license should be "SEE LICENSE IN ".